Source/WebCore/page/LocalFrameView.h

```diff
@@ -950,6 +952,8 @@ class LocalFrameView final : public FrameView {
     std::unique_ptr<ListHashSet<CheckedPtr<RenderEmbeddedObject>>> m_embeddedObjectsToUpdate;
     std::unique_ptr<WeakHashSet<RenderElement>> m_slowRepaintObjects;
 
+    HashMap<ScrollingNodeID, WeakPtr<ScrollableArea>> m_scrollingNodeIDToScrollableAreaMap;
```

Comment:
This deserved a comment saying "This is used for plugins", or, better, change the name so that's obvious:  `m_scrollingNodeIDToPluginScrollableAreaMap`